tests/dopamine/replay_memory/circular_replay_buffer_test.py

```diff
@@ -680,6 +680,38 @@ def testMemoryHasASpecifiedLock(self):
     memory = _create_dummy_memory(lock='lock-name')
     self.assertEqual(memory._lock, 'lock-name')
 
+  def testAddNodeToTrajectoryBuffer(self):
+    memory = circular_replay_buffer.OutOfGraphReplayBuffer(
+        observation_shape=OBSERVATION_SHAPE,
+        stack_size=STACK_SIZE,
+        replay_capacity=5,
+        batch_size=BATCH_SIZE,
+        contiguous_trajectories=True)
+    self.assertEqual(memory.cursor(), 0)
+    self.assertEqual(len(memory._trajectories), 0)
+    zeros = np.zeros(OBSERVATION_SHAPE)
+    memory.add(zeros, 0, 0, 0)
+    self.assertEqual(memory.cursor(), 0)
+    self.assertEqual(memory.add_count, 0)
+    self.assertEqual(len(memory._trajectories), 1)
+    self.assertEqual(memory._trajectory_lengths[0], 1)
+
+  def testAddTerminalNodeToTrajectoryBuffer(self):
```

Comment:
Any interest in testing not just that terminal nodes aren't added to the trajectory buffer, but that buffered states are moved to memory, too?

Reply:
This should be included in the new test I created `testNodeAddedToTrajectory`